### ISSUE TITLE
Add missing PHPDoc

### ISSUE THREAD

**Issue by arogachev** [yiisoft/yii-middleware#58]

**Comment by arogachev** [comment]
After #61 there are still places with missing PHPDoc, for example in `TrustedHostsNetworkResolver` (constructor, most of the private properties and methods). So let's keep this open for now.

**Comment by samdark** [comment]
Hmm... I'll do another PR then.

**Comment by samdark** [comment]
These aren't blocking release overall since public ones are what matters.
